numpy/add_newdocs.py

```diff
@@ -524,6 +550,13 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """)
 
+add_newdoc('numpy.core', 'nditer', ('close',
+    """
+    close()
+
+    Resolve all writeback semantics in writeable operands.
```

Comment:
Perhaps reference the `with` statement here.

Reply:
fixed (in doc-nditer PR)